libs/platform/src/lib/components/carosuel/slide.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, HostBinding, Input, ElementRef } from '@angular/core';
+
+
+@Component({
+    selector: 'fdp-carousel-item',
```

Comment:
The selector name should match the file and class names. Change `SlideComponent` and `slide.component.*` to `CarouselItemComponent` and `carousel-item.component.*`